pom.xml

```diff
@@ -1086,6 +1088,37 @@
                 <maven.compiler.release>${java.release}</maven.compiler.release>
             </properties>
         </profile>
+
+        <!--SBOM-->
+        <profile>
+            <!-- Generates SBOM. Skip with '-DskipSBOM'.-->
+                <id>sbom</id>
+                <activation>
+                  <property>
+                    <name>!skipSBOM</name>
+                  </property>
+                </activation>
+                <build>
+                    <plugins>
+                        <plugin>
+                            <groupId>org.cyclonedx</groupId>
+                            <artifactId>cyclonedx-maven-plugin</artifactId>
+                            <configuration>
```

Comment:
```suggestion
                            <groupId>org.cyclonedx</groupId>
                            <artifactId>cyclonedx-maven-plugin</artifactId>
                            <version>${version.plugin.cyclonedx}</version>
                            <configuration>
```
Need to use the version property set earlier in the file. Otherwise the property is ignored, it will pick the latest version and you get this warning:

```
[WARNING] Some problems were encountered while building the effective model for org.eclipse.microprofile:microprofile-tck-bom:pom:2.9-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.cyclonedx:cyclonedx-maven-plugin is missing. @ org.eclipse.microprofile:microprofile-parent:2.9-SNAPSHOT, /home/andrew/git/microprofile-parent/pom.xml, line 1103, column 33
[WARNING] 
[WARNING] Some problems were encountered while building the effective model for org.eclipse.microprofile:microprofile-parent:pom:2.9-SNAPSHOT
[WARNING] 'build.plugins.plugin.version' for org.cyclonedx:cyclonedx-maven-plugin is missing. @ line 1103, column 33
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```

Reply:
This change is also needed on the main branch.

Reply:
It was on the main already and it works there,

Reply:
When I build main, I see `2.8.0` being used and I get a similar warning. I've opened #86 to fix the problem on the main branch.